.github/workflows/move-to-done.yaml

```diff
@@ -0,0 +1,80 @@
+name: move-to-done
+
+# FYI secrets.ADD_TO_PROJECT_PAT is a personal access token belonging to @akeller.
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  move-to-done:
+    if: github.event.pull_request.merged == true
```

Comment:
This prevents us from running this action against PRs that are `closed` instead of `merged`. Is that correct? Should we move the closed ones to Done also?

Reply:
Good question. I move things to `Done` when there is no remaining action for that item. With that thinking, `closed` should be moved to `Done` as well. That said, we don't have many that are closed these days other than things you happen to be testing. In that sense, I would say we can leave this as a todo, or if it's relatively quick, make the change in this PR. Up to you!